db/seeds.rb

```diff
@@ -15,3 +15,11 @@
 Diversities::SEED_DISABILITIES.each do |disability|
   Disability.find_or_create_by!(name: disability.name, description: disability.description)
 end
+
+Settings.personas.each do |persona|
```

Comment:
We should avoid seeding these into the production DB. Not sure of the best way to handle that though at this point.

Reply:
I have an idea.